packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-i18n/main/index.md

```diff
@@ -5,15 +5,32 @@ title: Updates to the Sign-In Widget I18n properties
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
-## Overview
+This guide covers how to specify i18n translation strings after your org upgrades to the Okta Identity Engine.
+
+---
+**Learning outcomes**
+
+Understand how to override text strings with Identity Engine i18n strings.
```

Comment:
Again, phrase in terms of actual outcome? For example, add something to the end of the sentence like "...to create localized sign-in widgets." or something?

Reply:
Fixed